src/wallet/load.cpp

```diff
@@ -151,7 +151,7 @@ void StartWallets(WalletContext& context, CScheduler& scheduler)
     if (context.args->GetBoolArg("-flushwallet", DEFAULT_FLUSHWALLET)) {
         scheduler.scheduleEvery([&context] { MaybeCompactWalletDB(context); }, std::chrono::milliseconds{500});
     }
-    scheduler.scheduleEvery([&context] { MaybeResendWalletTxs(context); }, std::chrono::milliseconds{1000});
+    scheduler.scheduleEvery([&context] { MaybeResendWalletTxs(context); }, std::chrono::milliseconds{1000 * 60});
```

Comment:
```suggestion
    scheduler.scheduleEvery([&context] { MaybeResendWalletTxs(context); }, 1min});
```

https://en.cppreference.com/w/cpp/chrono/operator%22%22min

Reply:
oh nice, thanks!